### PR TITLE
CAS-1137 [docs] Strings localization guide

### PR DESCRIPTION
**Pull request by samiuelson**
https://stream-io.atlassian.net/browse/CAS-1137

### 🎯 Goal

Create a guide showing how to provide SDK strings localization.

### 🧪 Testing

1. Run docs:`npx stream-chat-docusaurus -i -s` 
2. Explore `/chat/docs/sdk/android/ui/guides/custom-translations/` page

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS (task created))
- [x] Reviewers added

### 🎉 GIF

[Joe tries hard speaking french](https://youtu.be/kujMmdwGPoo)

### CODE REVIEW

File: docusaurus/docs/Android/03-ui/05-guides/04-custom-translations.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Localizing SDK strings
+
+Stream SDK is available with US English string resources out-of-the box. The solution for adding you own localization is based on standard [Android mechanism](https://developer.android.com/guide/topics/resources/localization) of switching resources on system Locale change. You can provide custom localization for the SDK's string resources by overriding them in the locale-specific `/res/values` directories of your project.
```
Comment (bychkovdmitry) [inline docusaurus/docs/Android/03-ui/05-guides/04-custom-translations.md:3]:
```suggestion
Stream SDK is available with US English string resources out-of-the-box. The solution for adding your own localization is based on standard [Android mechanism](https://developer.android.com/guide/topics/resources/localization) of switching resources on system Locale change. You can provide custom localization for the SDK's string resources by overriding them in the locale-specific `/res/values` directories of your project.
```

File: docusaurus/docs/Android/03-ui/05-guides/04-custom-translations.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+:::note
+All of the string resources names provided by Stream SDK are prefixed with `stream_ui_`, e.g. `<string name="stream_ui_message_list_empty">No messages</string>`.
+:::
+
+### Setting up custom translations
+
+In this guide we are going to implement a custom translation for ChannelListHeaderView UI component. We are going to provide Polish language translation.
+
+Usually, base string resources are located in the `/res/values/strings.xml` file. In order to add a translations for the new language (PL) we are going to create a new `strings.xml` file under `res/values-pl` directory.
```
Comment (bychkovdmitry) [inline docusaurus/docs/Android/03-ui/05-guides/04-custom-translations.md:14]:
```suggestion
Usually, base string resources are located in the `/res/values/strings.xml` file. In order to add translations for the new language (PL) we are going to create a new `strings.xml` file under `res/values-pl` directory.
```

File: docusaurus/docs/Android/03-ui/05-guides/04-custom-translations.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+* [strings_mention_list.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_mention_list.xml) 
+* [strings_message_input.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_message_input.xml)
+* [strings_message_list.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_message_list.xml) 
+* [strings_message_list_header.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_message_list_header.xml) 
+* [strings_search.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_search.xml) 
+* [strings_common.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components/src/main/res/values/strings_common.xml)
+* [strings.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android/src/main/res/values/strings.xml)
```
Comment (bychkovdmitry) [inline docusaurus/docs/Android/03-ui/05-guides/04-custom-translations.md:48]: This is for the old UI components, I would remove this line

**Review by bychkovdmitry — APPROVED** [review]

File: docusaurus/docs/Android/03-ui/05-guides/04-custom-translations.md (hunk 3 above)
Comment (samiuelson) [reply docusaurus/docs/Android/03-ui/05-guides/04-custom-translations.md]: Sure, removed.

**Pull request merged**